dev/com.ibm.ws.security.social/src/com/ibm/ws/security/social/web/EndpointServices.java

```diff
@@ -106,7 +106,7 @@ void handleSocialLoginRequest(HttpServletRequest request, HttpServletResponse re
                     doRedirect(request, response, config);
                 }
             } catch (Exception e) {
-                throw new SocialLoginException("ERROR_PROCESSING_REDIRECT", null, new Object[] { e });
+                throw new SocialLoginException("ERROR_PROCESSING_REDIRECT", null, new Object[] { e.getMessage() });
```

Comment:
This one should stay as `e` since we don't know what kind of Exception we're catching here.